whois-api/src/test/java/net/ripe/db/whois/api/sso/AuthServiceClientTestIntegration.java

```diff
@@ -17,6 +18,7 @@
 @Tag("IntegrationTest")
 public class AuthServiceClientTestIntegration extends AbstractIntegrationTest {
 
+    private final static String UUID = "8ffe29be-89ef-41c8-ba7f-0e1553a623e5";
```

Comment:
Use a canonical order for modifiers: `private static final`

Ref. https://docs.oracle.com/javase/8/docs/api/java/lang/reflect/Method.html